Fix: typos (#1238)
l2beat/l2beat: packages/frontend/README.md

##### Known issues

> You should not import any files that depends on node related modules (e.g. fs) in stories. That will cause a lot of pain in searching why storybook is not working.

Reviewer: ```suggestion
You should not import files that depend on node-related modules (e.g. fs) in stories. That will cause much pain searching for why the storybook is not working.
```
